src/components/Card/Card.js

```diff
@@ -0,0 +1,13 @@
+import { CardContainer } from '../UI/CardContainer.styled';
+import { Paragraph } from '../UI/Paragraph.styled';
+import { HeadlineTwo } from '../UI/HeadlineTwo.styled';
+
+export default function Card({ entries }) {
+	return (
+		<CardContainer>
+			<Paragraph>{entries.category}</Paragraph>
+			<HeadlineTwo>{entries.name}</HeadlineTwo>
```

Comment:
I am confused by the logical order: A headline should be above the text it belongs to, shouldn't it?

Reply:
I was also not sure if I could do it that way. I have taken the headline, because it is like the headline of the card, but the category as a p-tag should be above it. Would you recommend to use a p-tag instead of the headline?
The sketch from my user story perhaps shows better how i mean it:
<img width="363" alt="Single entry" src="https://user-images.githubusercontent.com/98806457/168262748-3bc8b2dd-0e85-4ff8-b540-73ab6fa17b31.png">

Reply:
I agree that you use a headline, I was just wondering about the order. I would still suggest to put a headline above everything, but this is rather a design question. Maybe you can leave it this way and check for accessibility before merging.

Reply:
What do you mean by check for accessibility before merging? Checking with a screen reader?
It would also make sense to read Name (Headline), Kategorie and Adresse.
But yes, I did this for design reasons.